ghost/member-events/lib/MemberSubscriptionCancellationEvent.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Noticed while adding this, we also have [MemberPaidCancellationEvent](https://github.com/TryGhost/Ghost/blob/7084217d3df39fbf6f3cba302c63414282882873/ghost/member-events/lib/MemberPaidCancellationEvent.js) which was added for Analytics but doesn't seem to have any active usecases. Maybe a good candidate for removing in future to avoid having two similar named events

Reply:
How do you feel about `SubscriptionCancelledEvent` to match with `SubscriptionCreatedEvent`?

Reply:
Sounds good if we are moving to `SubscriptionCreatedEvent` above :)